components/DownloadButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react'
+import { AntDesign as Icon } from '@expo/vector-icons'
+import * as FileSystem from 'expo-file-system'
+
+import { useMeditation, useAppDispatch } from '../hooks'
+import { filepaths } from '../redux/meditationSlice'
+import { store } from '../redux/store'
+import { useFiles } from '../hooks/useFiles'
+
+export default function DownloadButton(props: any) {
+  const id = props.id
+  const meditation = useMeditation(id)
+  const files = useFiles('.mp3')
+  const uri = meditation?.uri || ''
+  const [audioFiles, setAudioFiles] = useState<string[]>([])
+  const [downloaded, setDownloaded] = useState(false)
+  const dispatch = useAppDispatch()
+
+  const filename = (path: string) => {
+    let filename = path.split('/').pop()
+    if (!filename) {
```

Comment:
We have a ton of lint warnings saying:

```
'filename' is already declared in the upper scope on line 19 column 9 
```

What if for local variables that shadow another variable we prefix them with a `_`? For example, `_filename`